base/doc/ltnews35.tex

```diff
@@ -218,6 +218,71 @@ \section{The \pkg{latex-lab} bundle}
 to drop the \texttt{testphase} key setting.
 
 
+\section{A new mark mechanism for \LaTeX{}}
+
+The mark mechanism is \TeX{}'s way to pass information to the
+page-building process which happens asychronously in order to
+communicate relevant data for running headers and footers to the
+latter, e.g., what is the first section on the page or the last
+subsection, etc. However, marks may also be
+used for other purposes. The new kernel module provides a generalized
+mechanism for marks of independent classes.
+
+The \TeX{} engines offer a low-level mark mechanism to
+communicate information about the content of the current page to
+the asynchronous operating output routine. It works by placing
+\cs{mark} commands into the source document.
+
+This mechanism works well for simple formats (such as plain \TeX)
+whose output routines are only called to generate pages. It
+fails, however, in \LaTeX{} (and other more complex formats),
+because here the output routine is sometimes called without
+producing a page, e.g., when encountering a float and placing it
+into one of the float regions.
+%
+When that happens \TeX{}'s  \cs{topmark} no
+longer reflects situation at the top of the next page when that
+page is finally boxed.
+
+Furthermore \TeX{} only offered a single mark while \LaTeX{} wanted to
+keep track of more than one piece of information.  For that reason
+\LaTeX{} implemented its own mark mechanism where the marks always
+contained two parts with their own interfaces: \cs{markboth} and
+\cs{markright} to set marks and \cs{leftmark} and \cs{rightmark} to
+retrieve them.
+
+Unfortunately, this extended mechanism (while supporting scenarios such
+as chapter/section marks) was far from general. The mark
+situation at the top of a page (i.e., \cs{topmark}) remained
+unusable and the two marks offered were not really independent of
+each other because \cs{markboth} (as the name indicates) was
+always setting both.
+
+
+The new mechanism now available in \LaTeX{} starting with the 2022
+release overcomes both issues:
+\begin{itemize}
+\item
+  It provides arbitrary many, fully independent named marks, that
+  can be allocated and from that point onwards used.
+\item
+  It offers access for each such marks to to retrieve its, top,
+  first and bottom value.
```

Comment:
"value" → "values"

Reply:
not sure about that one I think it should be singular, I leave that to copyediting by Karl later.